tests/test_external_account.py

```diff
@@ -26,6 +26,8 @@
 from google.auth import transport
 
 
+METRICS_HEADER_VALUE = "gl-python/3.7 auth/1.1 auth-request-type/at cred-type/imp"
```

Comment:
nit: Can this be named IMPERSONATE_ACCESS_TOKEN_REQUEST_METRICS_HEADER_VALUE to be consistent with the AWS tests and also to make it more clear this is only expected for SA impersonation requests?

Reply:
updated the name